### PR TITLE
Fix windows build

### PR DESCRIPTION
**Pull request by DominikDary**
## High-level description

This is fixing the windows build by using bzip2 mode for creating a tar file on windows.

## Corresponding DC/OS tickets (required)

  - [/DCOS_OSS-5499](https://jira.mesosphere.com/browse/DCOS_OSS-5499) Fix Teamcity CI/CD build

## Checklist for component/package updates:

If you are changing components or packages in DC/OS (e.g. you are bumping the sha or ref of anything underneath `packages`), then in addition to the above please also include:

  - [ ] Change log from the last version integrated (this should be a link to commits for easy verification and review): [example](https://github.com/dcos/dcos-mesos-modules/compare/f6fa27d7c40f4207ba3bb2274e2cfe79b62a395a...6660b90fbbf69a15ef46d0184e36755881d6a5ae)
  - [ ] Included a test which will fail if code is reverted but test is not. If there is no test please explain.
  - [ ] Test Results: [link to CI job test results for component]
  - [ ] Code Coverage (if available): [link to code coverage report]

### CODE REVIEW

**Comment by DominikDary** [comment]
@mesosphere-mergebot changelog-not-required Fixing windows build.

**Comment by DominikDary** [comment]
@mesosphere-mergebot label Ready For Review

**Comment by DominikDary** [comment]
@mesosphere-mergebot request-review jongiddy

**Comment by DominikDary** [comment]
@mesosphere-mergebot request-review  timaa2k

**Comment by timaa2k** [comment]
Please remove the comment as it is no longer true.

**Comment by DominikDary** [comment]
@mesosphere-mergebot bump-ee

File: pkgpanda/util.py [hunk 1]
```diff
@@ -385,11 +385,13 @@ def _tar_filter(tar_info: tarfile.TarInfo) -> tarfile.TarInfo:
...
 
 def make_tar(result_filename, change_folder):
-    # In the past we used bzip2 on Windows here. At the time of writing there is no
-    # test which fails on Windows. If this fails due to lzma not being
-    # available check liblzma linking on the DC/OS python package or consider
-    # using bzip2 instead.
-    with tarfile.open(name=str(result_filename), mode='w:xz') as tar:
+    tar_mode = ''
+    if is_windows:
+        tar_mode = 'w:bz2'
```
Comment (orsenthil) [inline pkgpanda/util.py]: Interesting change. :)

**Pull request merged**